org.lflang/src/org/lflang/generator/c/CCoreFilesUtils.java

```diff
@@ -39,6 +39,13 @@ public static List<String> getCTargetSrc() {
         );
     }
 
+    public static List<String> getArduinoTargetHeaders() {
+        return List.of(
+            "Arduino.h",
```

Comment:
I believe all our files are in lower case. Let's try to be consistent and make this lowercase as well.